Formula/xapian.rb

```diff
@@ -101,6 +92,13 @@ def install
   end
 
   def caveats
+    if build.with? "python"
```

Comment:
This isn't about that; there's a change we're still tracking down in how the latest versions of SWIG & some stuff we do interact which means we need to augment the `PYTHONPATH`. Since this is grotty (and I'm only just becoming aware of the detail of the problem), I'll understand if you'd rather wait until we figure out a proper fix for it and release 1.4.1.